client/src/i18n/en/table.json

```diff
@@ -23,6 +23,7 @@
             "CHARGE": "Charge",
             "CLIENT": "Client",
             "CODE": "Code",
+            "COMMENT" :"Commentaire",
```

Comment:
This key should already be translated somewhere, right?  Can we just reuse the old key?